core/src/main/java/com/datastrato/gravitino/storage/relational/po/SecurableObjectPO.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.storage.relational.po;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Preconditions;
+
+public class SecurableObjectPO {
```

Comment:
Again, do you need a securable object id, can you please think a bit on this?

Reply:
Securable object id seems not useful after think twice.